sanicbot/core/cogs.py

```diff
@@ -0,0 +1,58 @@
+import re
+
+import httpx
+from discord import Message
+from discord.ext import commands
+from discord.ext.commands import Context
+
+from sanicbot.core.utils import failure_message, success_message
+
+
+class GitCog(commands.Cog):
+    issue_pattern = re.compile(r"#(?P<issue_id>[1,2]\d{3})")
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    async def lookup(self, ctx: Context, number: int, repo: str):
+        """
+        Lookup the issue and check if it exists within a repo found in the Sanic organization. Redirection to
+        retrieve pull request is automatic due to the fact that Github issues them sequentially.
+
+        :param ctx: Discord context.
+
+        :param number: Issue number.
+
+        :param repo: Repository of issue being looked up.
+
+        :return: url, response_code
+        """
+        url = f'https://github.com/sanic-org/{repo}/issues/{number}'
```

Comment:
Assuming  we have not yet run `black`

Reply:
Black is not yet run no, and docketfile isnt really necessary due to the simplicity of the project. It's no issue just running python3 sanic.py and call it a day.

Reply:
I dont think a dockerfile is needed and no `black` has not been run yet. I'll clean up the requirements.txt file now.

Reply:
I think we should build an image. It will make deployment easier. But that can be a different PR.